pkg/epp/controller/inferencepool_reconciler_test.go

```diff
@@ -21,6 +21,9 @@ import (
 	"testing"
 	"time"
 
+	v1 "sigs.k8s.io/gateway-api-inference-extension/api/v1"
+	"sigs.k8s.io/gateway-api-inference-extension/apix/v1alpha2"
```

Comment:
I think golangci-lint is going to complain about the placement of these imports - should be grouped with other imports from same package.